cns/NetworkContainerContract.go

```diff
@@ -81,7 +81,8 @@ const (
 	// Delegated VM NICs are projected from VM to container network namespace
 	DelegatedVMNIC NICType = "DelegatedVMNIC"
 	// BackendNIC NICs are used for infiniband nics on a VM
-	BackendNIC NICType = "BackendNIC"
+	BackendNIC  NICType = "BackendNIC"
+	AccelnetNIC NICType = "AccelnetNIC"
```

Comment:
should this be BackendNIC_Accelnet? I see FrontendNIC_Accelnet in DNC

Reply:
these are the NIC-type names discussed in swift2.0+singularity forum & I guess we should adhere to these throughout the flow from DNC-RC -> DNC -> CNS, can we align and use these for use in CNS as well? 
![image](https://github.com/Azure/azure-container-networking/assets/26335310/65f323cc-8396-4bfd-951d-8a648c1a2fdd)

I'm seeing that we'd have to pass this from the value stored in `NodeNetworkInterfaceTable` to DNC & hence set the value in MTPNC to be used by CNS for programming various NIC-types. 
cc @thatmattlong @shreyashastantram @prmathur-microsoft

Reply:
agree! we should stick to one naming convention.

Reply:
Should we also change the `InfraNIC` & `DelegatedVMNIC` into `NodeNetworkInterfaceInfraNIC` & `NodeNetworkInterfaceDelegatedVMNIC`?